### PR TITLE
Show name of person who is answering the question

### PR DESCRIPTION
**Pull request by nwalters512**
- Show name of person who is answering a question
- Hide the "Finish answering" button for course staff that aren't answering the given question

I'm not sure if the green stripe is a good idea or not. Thoughts?

screen shot 2018-02-26 at 4 42 16 pm

### CODE REVIEW

File: models/Question.js [hunk 1]
```diff
@@ -20,17 +20,30 @@ module.exports = (sequelize, DataTypes) => {
...
     models.Question.belongsTo(models.User, { as: 'answeredBy' })
+
+    models.Question.addScope('defaultScope', {
+      attributes: {
+        include: ['askedById', 'answeredById', 'queueId'],
+      },
+      include: [{
+        model: models.User,
+        as: 'answeredBy',
+        /* attributes: {
```
Comment (genevievehelsel) [inline models/Question.js]: delete this comment?

File: api/questions.js [hunk 2]
```diff
@@ -11,17 +11,16 @@ const { requireQueue, requireQuestion, failIfErrors } = require('./util')
...
+/* eslint-disable no-param-reassign */
+function modifyBeingAnswered(question, answering) {
   question.beingAnswered = answering
   if (answering) {
     question.answerStartTime = new Date()
   } else {
     question.answerEndTime = new Date()
   }
-
-  return question.save()
```
Comment (genevievehelsel) [inline api/questions.js]: why don't you need a save here?

**Comment by genevievehelsel** [comment]
p.s. i like the green bar 😄

File: api/questions.js (hunk 2 above)
Comment (nwalters512) [reply api/questions.js]: This makes the function a bit more specific in its functionality. Take the `POST /` route: I also want to modify the `answeredById` attribute before saving.

Comment (genevievehelsel) [reply api/questions.js]: ahhhh gotcha :)

**Pull request merged**